components/dashboard/src/teams/NewTeam.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { FormEvent, useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { getGitpodService } from "../service/service";
+import { UserContext } from "../user-context";
+
+export default function () {
+    const { user, setContext } = useContext(UserContext);
+    const history = useHistory();
+
+    const [ creationError, setCreationError ] = useState<Error>();
+    let name = '';
+    const createTeam = async (event: FormEvent) => {
+        event.preventDefault();
+        try {
+            if (!user) {
+                throw new Error('Not signed in');
+            }
+            const team = await getGitpodService().server.createTeam(name);
+            const teams = await getGitpodService().server.getTeams();
+            setContext({ user, teams });
+            history.push(`/${team.slug}`);
+        } catch (error) {
+            console.error(error);
+            setCreationError(error);
+        }
+    }
+    return <div className="flex flex-col w-96 mt-16 mx-auto items-center">
+        <h1>New Team</h1>
+        <p className="text-gray-500 text-center text-base">Teams allow you to <strong>group multiple projects</strong>, <strong>collaborate with others</strong>, <strong>manage subscriptions</strong> with one centralized billing, and more. <a href="">Learn more</a></p>
+        <form className="mt-8 w-full" onSubmit={createTeam}>
+            <div className="border rounded-xl p-6 border-gray-100">
+                <h3 className="text-center text-xl mb-6">What's your team's name?</h3>
+                <h4>Team Name</h4>
+                <input className="w-full" type="text" onChange={event => name = event.target.value} />
+                {!!creationError && <p className="text-gitpod-red">{creationError.message}</p>}
```

Comment:
**issue**: Ideally, this would also change the input border to red but should be fine to leave this as is for this iteration. We'll get back to input validation styles.

Reply:
Added an `error` class for some inputs. Let's see how this looks. 😁